### PR TITLE
new 'build & run' instructions

### PR DESCRIPTION
**Pull request by llama-0**
- added docker-compose.yml
- changed MongoDB port to default
- modified README.md

### CODE REVIEW

**Comment by cloudrac3r** [comment]
IIRC, `docker-compose up` should automatically build first if needed.

File: WebApp/WebAppConfig_dockerized.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     },
     "DocUrl": "https://github.com/caphindsight/TrulyQuantumChess/wiki",
     "Captcha": {
-        "Enabled": true,
+        "Enabled": false,
```
Comment (caphindsight) [inline WebApp/WebAppConfig_dockerized.json:23]: This is the production config, captha needs to stay on.

File: WebAppConfig_dockerized.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+﻿{
```
Comment (caphindsight) [inline WebAppConfig_dockerized.json:1]: You may want to put it into the same directory with the other configs, and perhaps rename to something like `WebAppConfig_dockerized_local.json` to keep it distinct from the production config.

**Comment by llama-0** [comment]
> IIRC, `docker-compose up` should automatically build first if needed.

With `docker-compose up` I was under the impression that you need `--build` flag for this, but may be I didn't get what you mean 🤔

File: Dockerfile.local [hunk 3]
```diff
@@ -0,0 +1,11 @@
+FROM mono:latest
+EXPOSE 9000
+COPY ./ /src
+WORKDIR /src
+RUN nuget restore && \
+    find . -name "*RuntimeInformation.dll" -delete && \
+    xbuild /p:Configuration=Release && \
```
Comment (caphindsight) [inline Dockerfile.local:7]: `Configuration=Debug` is probably better for running locally.

**Comment by caphindsight** [comment]
@llama-0 it seems this project is still generating attention. I have hardy been spending any time on GitHub lately, so if you wanted to step in and maintain it I’d be down. You are already a collaborator so you can make changes and review pull requests, feel free to assume ownership.

**Pull request merged**